src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -855,6 +856,8 @@ ov::Any Engine::get_ro_property(const std::string& name, const ov::AnyMap& optio
         return decltype(ov::intel_cpu::denormals_optimization)::value_type(engConfig.denormalsOptMode == Config::DenormalsOptMode::DO_On);
     } else if (name == ov::intel_cpu::sparse_weights_decompression_rate) {
         return decltype(ov::intel_cpu::sparse_weights_decompression_rate)::value_type(engConfig.fcSparseWeiDecompressionRate);
+    } else if (name == ov::execution_devices) {
+        return decltype(ov::execution_devices)::value_type{ov::IPlugin::get_device_name()};
```

Comment:
sorry, I didn't make it clear, I mean  use get_device_name() instead of ov::IPlugin::get_device_name()
```suggestion
        return decltype(ov::execution_devices)::value_type{get_device_name()};
```

Reply:
Oh I see, done